build.sbt

```diff
@@ -790,6 +790,24 @@ lazy val ast = (project in file("lib/scala/ast"))
     Compile / sourceGenerators += GenerateAST.task,
   )
 
+lazy val parser = (project in file("lib/scala/parser"))
+  .settings(
+    fork := true,
+    javaOptions += {
+      val root = baseDirectory.value.getParentFile.getParentFile.getParentFile
+      s"-Djava.library.path=$root/target/rust/debug"
+    },
```

Comment:
I understand that currently no projects depend on the `parser`, but in the future, at least `runtime` will, right?

Is there a plan on how to include these binary dependencies in the distribution of `runtime`? Will they be packaged in some `native_lib` directory or inside the JAR? How should users/programs launching the `runtime` handle the `java.library.path`?

I feel like this should be documented. I guess it may be added when `runtime` really becomes dependent on the new `parser`, but I'm worried we may forget to do so then, so maybe at least a general plan on how this will happen would be good?

Reply:
This should definitely be documented now, both in the parser documentation (the why), and the distribution documentation (the what).